lib/iris/coords.py

```diff
@@ -2228,14 +2256,26 @@ def nearest_neighbour_index(self, point):
         return result_index
 
     def xml_element(self, doc):
-        """Return a DOM element describing this Coord."""
+        """
+        Create the :class:`xml.dom.minidom.Element` that describes this
+        :class:`Coord`.
+
+        Args:
+
+        * doc:
+            The parent :class:`xml.dom.minidom.Document`.
+
+        Returns:
+            The :class:`xml.dom.minidom.Element` that will describe this
+            :class:`DimCoord`, and the dictionary of attributes that require
+            to be added to this element.
+
+        """
         # Create the XML element as the camelCaseEquivalent of the
         # class name
         element = super().xml_element(doc=doc)
 
-        element.setAttribute("points", self._xml_array_repr(self.points))
```

Comment:
@bjlittle can you just confirm this line supposed to be deleted? comment below suggests that it is

Reply:
@jamesp Yup, that's fine, and it's a good question...

It's actually redundant, and I only noticed it by working through the inheritance behaviour. 

`Coord.xml_element` calls `_DimensionalMetadata.xml_element`, and in the super class it performs the following:
```python
        # The values are referred to "points" of a coordinate and "data"
        # otherwise.
        if isinstance(self, Coord):
            values_term = "points"
        else:
            values_term = "data"
        element.setAttribute(values_term, self._xml_array_repr(self._values))
```
So as it stood, the `_DimensionalMetadata` was setting the `points` attribute, then the `Coord` came along an stomped over it with the same result... seemed like worthwhile trimming that fat whilst we're here.

If it didn't work... we'd have test failures all over the place. So I believe this is fine.

Lovely spot though :eagle: :eyes: